src/index.ts

```diff
@@ -78,9 +78,20 @@ export type BunPluginHTMLOptions = {
 	 * Processes the files before they are processed by `bun-plugin-html`. Useful for things like tailwindcss.
 	 */
 	preprocessor?: (processor: Processor) => void | Promise<void>;
+	/**
+	 * Replace path strings
+	 */
+	replacePathStrings?: boolean;
```

Comment:
Can we get a little more detail on what exactly changes when you enable this?

Reply:
add test cases on keepPathStrings, using **keepPathStrings** instead of replacePathStrings